pkg/cli/run.go

```diff
@@ -0,0 +1,92 @@
+package cli
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/mattn/go-isatty"
+	"github.com/replicate/cog/pkg/docker"
+	"github.com/replicate/cog/pkg/logger"
+	"github.com/replicate/cog/pkg/util/terminal"
+	"github.com/spf13/cobra"
+)
+
+func newRunCommand() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "run <command> <args>",
+		Short: "Run a command inside a Docker environment",
+		RunE:  run,
+		Args:  cobra.MinimumNArgs(1),
```

Comment:
@bfirsh should this not be `2`?

Reply:
Nope, I don't think so. It just needs a command to run. Usage string could perhaps be clearer.